ksqldb-common/src/main/java/io/confluent/ksql/reactive/BasePublisher.java

```diff
@@ -37,6 +37,8 @@
   private Subscriber<? super T> subscriber;
   private long demand;
   private boolean cancelled;
```

Comment:
I think cancelled can be removed completely now?

Reply:
A user can still call `cancel()` on the subscription right? If this happens, subsequent calls to `BufferedPublisher#accept()` will not actually add to the buffer.